mobile_back_samsung/samsung_backend.mk

```diff
@@ -16,4 +16,39 @@
 ifeq (${WITH_SAMSUNG},1)
   $(info WITH_SAMSUNG=1)
   android_samsung_backend_bazel_flag=--//android/java/org/mlperf/inference:with_samsung="1"
+
+  MOBILE_BACK_SAMSUNG_LIB_ROOT=$(shell echo mobile_back_samsung/samsung_backend/lib* | awk '{print $$NF}')
+  $(info MOBILE_BACK_SAMSUNG_LIB_ROOT=${MOBILE_BACK_SAMSUNG_LIB_ROOT})
+
+  ifeq (${MOBILE_BACK_SAMSUNG_LIB_ROOT},)
+    $(error MOBILE_BACK_SAMSUNG_LIB_ROOT env must be defined when building with samsung backend)
+  endif
+  $(info MOBILE_BACK_SAMSUNG_LIB_ROOT=${MOBILE_BACK_SAMSUNG_LIB_ROOT})
+  backend_samsung_docker_args=-v "${MOBILE_BACK_SAMSUNG_LIB_ROOT}:/mnt/samsung_backend" --env MOBILE_BACK_SAMSUNG_LIB_ROOT=/mnt/samsung_backend
+  backend_samsung_android_files= \
+    ${BAZEL_LINKS_PREFIX}bin/android/commonlibs/lib_arm64/libc++_shared.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libsamsungbackend.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_model.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_profiler.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_rt.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_ud_gpu.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_ud_cpu.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_ud_npu.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_nn.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_osal.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_xtool.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libc++.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libeden_nn_on_system.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libutils.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libcutils.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libvndksupport.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libbase.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libcgrouprc.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libhidlbase.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libgpu_boost_vendor.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libprocessgroup.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libvendor_samsung_slsi_hardware_gpu_boost.so
+  # main binaries for Samsung backend are prebuilt
+  backend_samsung_android_target=//android/commonlibs:commonlibs
+  backend_samsung_filename=libsamsungbackend
```

Comment:
All of this was removed in `android-v2` branch because it's only used for Flutter build.